Tests/StencilTests/NodeSpec.swift

```diff
@@ -25,6 +25,41 @@ final class NodeTests: XCTestCase {
       let node = TextNode(text: "Hello World")
       try expect(try node.render(self.context)) == "Hello World"
     }
+    it("Trims leading whitespace") {
+      let text = "      \n Some text     "
```

Comment:
why not to use multiline strings? they seem to be more helpful to visualise behaviour. applicable to other places with single line strings.